styles.css

```diff
@@ -0,0 +1,25 @@
+:root{
+    --primary-colour:blue;
+    --secondary-coluor:#44686e;
+}
+
+img {
+    width: 100 display: block height: 70px;
```

Comment:
Please, let all your comments include an explanation, a "why", for each requested change. This will help them see the reasons with you. Some reasons might seem obvious to you but not to the author of the code.